packages/icon/src/template.ts

```diff
@@ -93,11 +100,11 @@ module.exports = function template(
       return ${jsx};
     }
 
-    ${componentName}.displayName = ${componentName};
+    ${componentName}.displayName = 'Glyph' + '${componentName}';
```

Comment:
Let's remove the runtime cost.

```suggestion
    ${componentName}.displayName = 'Glyph${componentName}';
```

Reply:
👍

Reply:
Changing this causes a Babel error: `Unknown substitution "$5" given`

Reply:
I wonder if you have to escape the characters. Or perhaps use a template string?

Reply:
This is actually unnecessary now, because we are checking for isGlyph rather than needing to prefix the displayName as such. So reverted this back to '${componentName}'